templates/vdcuser/vdcuser.py

```diff
@@ -34,6 +34,16 @@ def get_fqid(self):
         return "%s@%s" % (self.name, provider) if provider else self.name
 
     def install(self):
+        '''
+        Install vdcuser
+        '''
+
+        try:
+            self.state.check('actions', 'install', 'ok')
```

Comment:
If would be nicer if we could do the following:
```python
if self.state.check('actions', 'install', 'ok', die=False):
    return
```

Reply:
> `if self.state.get('actions','install') == {'install':'ok'}`

errors if `self.state == {}`